src/components/example/EDialog.tsx

```
@@ -0,0 +1,55 @@
+import { Button } from "@/components/ui/button"
```

ESLint

[error] Replace `"@/components/ui/button"` with `'@/components/ui/button'`

(prettier/prettier)